File: js/taller.js/29.js
let v1 = parseFloat(prompt("Número 1:"));
let v2 = parseFloat(prompt("Número 2:"));
let v3 = parseFloat(prompt("Número 3:"));

let menor, medio, mayor;

if (v1 <= v2 && v1 <= v3) {
    menor = v1;
    if (v2 <= v3) { [medio, mayor] = [v2, v3]; } 
    else { [medio, mayor] = [v3, v2]; }
} else if (v2 <= v1 && v2 <= v3) {
    menor = v2;
    if (v1 <= v3) { [medio, mayor] = [v1, v3]; } 
    else { [medio, mayor] = [v3, v1]; }
} else {
    menor = v3;
    if (v1 <= v2) { [medio, mayor] = [v1, v2]; } 
    else { [medio, mayor] = [v2, v1]; }
}

console.log(`Números en orden ascendente: ${menor}, ${medio}, ${mayor}`);

File: py/taller.py/29.py
n1 = float(input("Número 1: "))
n2 = float(input("Número 2: "))
n3 = float(input("Número 3: "))

# Lógica con condicionales para ordenar
if n1 <= n2 and n1 <= n3:
    menor = n1
    if n2 <= n3:
        medio, mayor = n2, n3
    else:
        medio, mayor = n3, n2
elif n2 <= n1 and n2 <= n3:
    menor = n2
    if n1 <= n3:
        medio, mayor = n1, n3
    else:
        medio, mayor = n3, n1
else:
    menor = n3
    if n1 <= n2:
        medio, mayor = n1, n2
    else:
        medio, mayor = n2, n1

print(f"Números en orden ascendente: {menor}, {medio}, {mayor}")